ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -1133,6 +1135,13 @@ extension BottomCommandingController: UITableViewDelegate {
         return configuredHeader
     }
 
+    public func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let section = expandedListSections[indexPath.section]
+        let item = section.items[indexPath.row]
+
+        return item.isHidden ? 0 : UITableView.automaticDimension
```

Comment:
This feels suspicious. Shouldn't we just not have a cell if an item is hidden? Setting the height to 0 sounds like it will just lead to accessibility bugs with accessible but zero-sized elements

Reply:
Updated the `heroItems` and `expandedListSections` arrays (i.e. the datasources) to filter out hidden items so that this approach isn't needed. The table view will now only try to render filtered items.